packages/react/src/components/pickers/PickerItem.types.ts

```diff
@@ -41,4 +42,10 @@ export interface IPickerItemProps<T> extends React.AllHTMLAttributes<HTMLElement
 
   /** Aria-label for the picked item remove button. */
   removeButtonAriaLabel?: string;
+
+  /**
+   *  Props for the icon used in the remove button.
+   *  @defaultvalue \{iconName:'Cancel'\}
```

Comment:
It's easier to read if this is just wrapped in backticks.
```suggestion
   *  @defaultvalue `{ iconName:'Cancel' }`
```